Source/WebKit/UIProcess/WebPreferences.cpp

```diff
@@ -197,11 +197,6 @@ void WebPreferences::updateBoolValueForKey(const String& key, bool value, bool e
         return;
     }
 
-#if ENABLE(WEB_PUSH_NOTIFICATIONS)
-    if (key == WebPreferencesKey::builtInNotificationsEnabledKey())
-        WebCore::DeprecatedGlobalSettings::setBuiltInNotificationsEnabled(value);
```

Comment:
How does this work exactly? Aren't there still a bunch of places where we're checking `DeprecatedGlobalSettings::builtInNotificationsEnabled()`?

Reply:
Hm ok, I see now that [288101@main](https://commits.webkit.org/288101@main) changed the implementation of `DeprecatedGlobalSettings::builtInNotificationsEnabled()`.